src/main/java/com/scopely/mapper/JsonNodeAttributeValueMapper.java

```diff
@@ -110,6 +114,12 @@ public static ObjectNode convert(Map<String, AttributeValue> map, ObjectMapper o
                 return setAVForArray(node, attributeValue);
             case MISSING:
             case BINARY:
+                try {
```

Comment:
This doesn't look right. Surely you don't mean to use this logic for the MISSING case.